src/AnalogClock.js

```diff
@@ -10,16 +10,26 @@ export default class AnalogClock extends Component {
     constructor(props) {
         super();
 
-        const date = new Date();
+        const date = this.initializeTime(props.gmtOffset);
         this.state = {
-            seconds: date.getSeconds(),
-            minutes: date.getMinutes(),
-            hour: date.getHours(),
+            seconds: date[0],
+            minutes: date[1],
+            hour: date[2],
         };
 
         this.styles = cssTransform(Styles, props);
     }
 
+    initializeTime(gmtOffset) {
+        const now = new Date();
```

Comment:
@zackargyle Thank you very much for your suggestion and I worked accordingly. How about this `gmtOffset`? Please review. Hope it would be use full.